### PR TITLE
add new task, alg5

### PR DESCRIPTION
**Pull request by marina-nazarenko**

### CODE REVIEW

File: src/task1/Alg5.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package task1;
+
+    public class Alg5 {
+        public static void main(String[] args) {
+            Alg5 alg5 = new Alg5();
+            alg5.replacement();
+
+        }
+
+        private void replacement() {
```
Comment (denis-tumel) [inline src/task1/Alg5.java:10]: rename method name

File: src/task1/Alg5.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
+package task1;
+
+    public class Alg5 {
+        public static void main(String[] args) {
+            Alg5 alg5 = new Alg5();
+            alg5.replacement();
+
```
Comment (denis-tumel) [inline src/task1/Alg5.java:7]: redurant line

Comment (marina-nazarenko) [reply src/task1/Alg5.java:7]: fixed

File: src/task1/Alg5.java (hunk 1 above)
Comment (marina-nazarenko) [reply src/task1/Alg5.java:10]: fixed

**Review by denis-tumel — APPROVED** [review]

**Pull request merged**